test/e2e/alterBuiltManifest.ts

```diff
@@ -0,0 +1,92 @@
+/**
+ * Before this file was created, there was no way to pass runtime flags from an E2E test to the built extension.
+ * (Well okay, more precisely, you could change certain things through Fixtures, and you could change certain things
+ * through Ganache parameters, but both of these didn't load until late in the app load. Sentry is one of the first
+ * things to load, and it needed a way to get runtime flags very early in the app load.)
+ *
+ * The way this works is a bit of a hack, but it's the only solution we could come up with that would load early enough:
+ *
+ * 1) A global beforeEach hook in the E2E tests backs up the manifest.json file
+ * 2) The helpers.withFixtures() function calls setManifestFlags(), which reads in the manifest file and parses it
+ * 3) We alter the manifest with CircleCI environment variables and custom flags, then write it back to manifest.json
+ * 4) The test runs, and the built extension can call getManifestFlags() to get the custom flags
+ * 5) A global afterEach hook restores the backup copy of the manifest, so that the next test gets the normal manifest
+ */
+import fs from 'fs';
+import { ManifestFlags } from '../../app/scripts/lib/manifestFlags';
+
+const folder = `dist/${process.env.SELENIUM_BROWSER}`;
+
+// Global beforeEach hook to backup the manifest.json file
+if (typeof beforeEach === 'function' && process.env.SELENIUM_BROWSER) {
```

Comment:
Nit: I think the `beforeEach`/`afterEach` logic and the `setManifestFlags` logic should be in two separate files. But I'm not going to try to hold up the PR for it.

Reply:
Hmm what would you call those two files?

Reply:
Maybe `manifest-flag-hooks.ts` (or in camelCase, what the files use in this folder) and `setManifestFlags.ts`? Naming things is too hard.

Reply:
Done